### ISSUE TITLE
Unknown node type: graphql.language.InputObjectTypeDefinition

### ISSUE THREAD

**Issue by abedeoan** [graphql-java-generator/graphql-maven-plugin-project#5]
I am having troubles generating code in client or server mode for input objects. 

This is my input: 
```
input _DataFilter {
    Domain: String
    Domain_in: [String!]
}
```
and here it's the error message from the plugin:
`Unknown node type: graphql.language.InputObjectTypeDefinition InputObjectTypeDefinition{name='_DataFilter', directives=[], inputValueDefinitions=[InputValueDefinition{name='Domain', type=TypeName{name='String'}, defaultValue=null, directives=[]}, InputValueDefinition{name='Domain_in', type=ListType{type=NonNullType{type=TypeName{name='String'}}}, defaultValue=null, directives=[]}]}`

Is there a limitation of the plugin around input objects in client mode? Or am I constructing the input wrong?

Any feedback would be appreciated.

**Comment by abedeoan** [comment]
Later edit: support for Input types will be added in a later version of the plugin.

**Comment by etienne-sf** [comment]
Yes.
:)

the support is commited in git. After some more test, I'll release a new version.

**Comment by abedeoan** [comment]
The first few tests look promising! Thank you, Etienne, for the fast reply.

I will perform a few more checks the upcoming days; if everything is OK, I will close down this Issue.

**Comment by etienne-sf** [comment]
Solved in the 1.2 version
